### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by PrideToons**
# New Port for { REDO! }

## Game Information
- **Title**: { REDO! }
- **URL**: { https://redo99.itch.io/redo }

### CFW Tests
Ensure your game has been tested on all major CFWs:
- [X] AmberELEC
- [X] ArkOS
- [X] ROCKNIX
- [X] muOS

### Resolution Tests
Test all major resolutions:
- [X] 480x320
- [X] 640x480
- [X] 720x720 (RGB30)
- [X] Higher resolutions (e.g., 1280x720)

## File Structure
- Your port should have the following structure:
  - portname/
    - port.json
    - README.md
    - screenshot.jpg
    - cover.jpg
    - gameinfo.xml
    - Port Name.sh
    - portname/
      - <portfiles here>

## Additional Resources
For an in-depth guide on creating a pull request, refer to: [PortMaster Game Packaging Guide](https://portmaster.games/packaging.html#creating-a-pull-request)

### CODE REVIEW

**Comment by JanTrueno** [comment]
Hey PrideToons. You cant just add the zip like this. See discord :)

File: ports/redo/REDO!.sh [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+get_controls
+[ -f "${controlfolder}/mod_${CFW_NAME}.txt" ] && source "${controlfolder}/mod_${CFW_NAME}.txt"
+
+$ESUDO chmod 666 /dev/tty0
+
+GAMEDIR=/$directory/ports/redo
+
+> "$GAMEDIR/log.txt" && exec > >(tee "$GAMEDIR/log.txt") 2>&1
+
+export LD_LIBRARY_PATH="/usr/lib32:$GAMEDIR/libs:$GAMEDIR/utils/libs"
```
Comment (Cebion) [inline ports/redo/REDO!.sh:28]: Ld_library_path is not correct. Check example scripts for the correct way to export it.

**Comment by PrideToons** [comment]
> Hey PrideToons. You cant just add the zip like this. See discord :)

Just changed it. Is it good enough?

File: ports/redo/cover.png
Comment (Cebion) [inline ports/redo/cover.png:1]: Cover has to show gameplay and the logo / box art. Else just the screenshot is used.

**Review by Cebion — CHANGES_REQUESTED** [review]

File: ports/redo.zip
[the remark this answers is not in the document]
Comment (PrideToons) [reply ports/redo.zip:1]: Is it still there? I thought I deleted it.

File: ports/redo/REDO!.sh (hunk 1 above)
Comment (PrideToons) [reply ports/redo/REDO!.sh:28]: I don't quite understand this one. I've seen that same line of code in Omega Strike and Super Omega Zero. Should I delete it maybe?

File: ports/redo/cover.png
Comment (PrideToons) [reply ports/redo/cover.png:1]: So just delete the Cover?

File: ports/redo/REDO!.sh (hunk 1 above)
Comment (JanTrueno) [reply ports/redo/REDO!.sh:28]: Omega strike looks like this: ```export LD_LIBRARY_PATH="/usr/lib32:$GAMEDIR/libs:$LD_LIBRARY_PATH"```

File: ports/redo/cover.png
Comment (PrideToons) [reply ports/redo/cover.png:1]: > Cover has to show gameplay and the logo / box art. Else just the screenshot is used.

I made some changes what do you think Cebion?

**Comment by PrideToons** [comment]
Alright, I changed the XML file Cebion!

**Review by kloptops — APPROVED** [review]

**Pull request merged**